operator/cmd/mesh/profile-diff.go

```diff
@@ -40,7 +40,7 @@ func profileDiffCmd(rootArgs *rootArgs, pfArgs *profileDiffArgs) *cobra.Command
 		Short: "Diffs two Istio configuration profiles",
 		Long:  "The diff subcommand displays the differences between two Istio configuration profiles.",
 		Example: `  # Profile diff by providing yaml files
-  istioctl profile diff manifests/profiles/default.yaml manifests/profiles/demo.yaml
+  istioctl profile diff --manifests manifests default demo
```

Comment:
```suggestion
  istioctl profile diff default demo -d manifests
```

Reply:
This makes it clear that we are diffing two profile from manifests folder.

Reply:
Both the commands work, right?

Reply:
Yes, both work fine. But I think we should use `command > subcommand > main flags > sub flags`.

Reply:
Does the description also need to be updated?
`# Profile diff by providing yaml files`

Reply:
Thanks, addressed comments.